cpp/src/arrow/filesystem/s3fs.cc

```diff
@@ -505,7 +509,6 @@ class WrappedRetryStrategy : public Aws::Client::RetryStrategy {
             detail, static_cast<int64_t>(attempted_retries)));
   }
 
- private:
```

Comment:
Nit: I think you can leave `s3_retry_strategy_` as private.

Reply:
Will do.